src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -71,5 +71,9 @@
         </service>
 
         <service id="services_resetter" class="Symfony\Component\HttpKernel\DependencyInjection\ServicesResetter" public="true" />
+
+        <service id="secret_var_processor" class="Symfony\Component\DependencyInjection\SecretVarProcessor" >
+            <tag name="container.env_var_processor" priority="-992" />
+        </service>
```

Comment:
I'm definitely open to other ideas here but this is how I'm allowing devs to "turn off" secrets by only overriding the enabled flag per environment. If the `SecretVarProcessor` is always loaded, but enabled is false, it won't have a `JweHandler` added to the definition in FrameworkExtension and will ignore the `secret:` prefix in config files, allowing the EnvVarProcessor to handle it however dev's managed secrets before (e.g., DotEnv). If I only load this conditionally, we will have to override any variables that have the `secret:` prefix whenever we override enabled, because they will appear to be unsupported. Does that seem okay or should I try to figure out a better way to do this?